connector/mqtt/locust/src/ejbca/certificate.py

```diff
@@ -112,9 +72,9 @@ def renew_cert(self) -> None:
         The procedure made here is described in:
         https://doc.primekey.com/ejbca/ejbca-operations/ejbca-operations-guide/ca-operations-guide/end-entities/certificate-renewal
         """
-        DojotAPI.reset_entity_status(self.jwt, self.c_name)
-        self.key = {"raw": "", "pem": self.generate_private_key()}
-        self.csr = {"raw": "", "pem": self.generate_csr()}
+        DojotAPI.reset_entity_status(self.jwt, self.crt["fingerprint"])
```

Comment:
You don't need this anymore, to renew a certificate, just ask for another one. The particularities of the EJBCA were internalized in the x509 identity component.
It would be interesting to revoke the previous certificate, but this would have no effect on the locust, unless there is a test scenario in which it is considered that the certificate is rejected by the Broker because it was revoked.

Review the file `cert_utils.py`, functions `revoke_cert(thing: Thing)` and `has_been_revoked(thing: Thing)`